src/uphelper.py

```
@@ -4,13 +4,18 @@
 from difflib import SequenceMatcher
+from typing import Any, Mapping, cast
 
 from cogs.redaction import redact_private_info
 from data.config import config
 from src.cleanup import cleanup, reset_terminal
 from src.console import console
 from src.trackersetup import tracker_class_map
 
+DEFAULT_CONFIG: Mapping[str, Any] = cast(Mapping[str, Any], config.get('DEFAULT', {}))
+if not isinstance(DEFAULT_CONFIG, dict):
+    raise ValueError("'DEFAULT' config section must be a dict")
```

Ruff (0.14.10)

Prefer `TypeError` exception for invalid type

(TRY004)